packages/eslint-plugin-query/package.json

```diff
@@ -51,5 +51,8 @@
     "@typescript-eslint/utils": "^5.41.0",
     "eslint": "^8.26.0",
     "tsup": "^6.3.0"
+  },
+  "peerDependencies": {
+    "eslint": "^6.0.0 || ^7.0.0 || ^8.0.0"
```

Comment:
shouldn't adding this result in a change in the our lockfile ?

Reply:
I'm not too sure how pnpm works, but:
- i've ran  `pnpm i --frozen-lockfile` before & after changes;
- from what i've observed - pnpm doesn't store `peerDependencies` for packages that are defined within repository. For example [packages/react-query doesn't have peerDependencies in lock file but has them defined within package.json](https://github.com/TanStack/query/blob/857ecd91b6d86306bacd3785e3e2f71d7287ad06/pnpm-lock.yaml#L1272C3-L1283C21). Which makes sense - you lock it from consumer end.

Reply:
you're right, there's no change. For the future, you'd want to run `pnpm i` to update to potentially update the lockfile.